de-lib/src/main/java/org/iplantc/de/client/services/DiskResourceServiceFacade.java

```diff
@@ -129,7 +131,7 @@ void moveContents(final Folder sourceFolder,
      * @param dest id of the destination folder.
      * @param callback service success/failure callback
      */
-    void importFromUrl(String url, DiskResource dest, AsyncCallback<String> callback);
+    void importFromUrl(String url, DiskResource dest, DECallback<String> callback);
```

Comment:
Good catch - I'll update the presenter that calls this endpoint to `AppsCallback` since that makes the most sense right now.  We can switch it to `DataCallback` when CORE-6214 goes through.